ios/PurchasesHybridCommon/PurchasesHybridCommonIntegrationTests/BaseIntegrationTests.swift

```diff
@@ -16,10 +16,6 @@ class BaseIntegrationTests: XCTestCase {
         return .default
     }
 
-    override class func setUp() {
-         BundleSandboxEnvironmentDetector.default = MockSandboxEnvironmentDetector()
```

Comment:
This is no longer necessary thanks to https://github.com/RevenueCat/purchases-ios/pull/2276